addons/form/nodes/Validator.gd

```diff
@@ -159,12 +154,22 @@ func validate(
 		predefined_regex_result = true
 
 	##-- user_regex --##
-	if user_regex not in ["", null, ".*"]:
+	if user_regex.compile(custom) == OK and user_regex.is_valid() \
+	and user_regex.get_pattern() not in ["", null, ".*"]:
```

Comment:
```suggestion
	and user_regex.get_pattern() not in REDUNDANT_PATTERN:
```
Could you define A const somewhere at the start of the file with this list, because it's used multiple times